persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/AlterationCountsMapper.xml

```diff
@@ -20,10 +20,11 @@
             FROM mutation_event
             INNER JOIN mutation ON mutation_event.MUTATION_EVENT_ID = mutation.MUTATION_EVENT_ID
             INNER JOIN gene ON mutation_event.ENTREZ_GENE_ID = gene.ENTREZ_GENE_ID
+            <include refid="fromIncludeCustomAnnotationsMutation"/>
             <where>
                 <choose>
                     <when test="mutationTypes.hasNone()">NULL</when>
-                    <when test="!mutationTypes.hasAll()">
+                    <when test="not mutationTypes.hasAll()">
```

Comment:
Are these the same? Why do we prefer one over the other?

Reply:
They are the same in meaning. I switched to `not` because in some instances the evaluation of `!` caused problems. I guess OGNL (the expression language used by MyBatis) can be a bit unpredictable in some evaluations.